apps/web/src/components/EmploymentEquity/EquityOptions.tsx

```diff
@@ -106,13 +106,12 @@ const EquityOptions = ({
   };
 
   const handleMultipleFieldSave = (data: UpdateUserAsUserInput) => {
-    onUpdate(data)
-      .then(() => {
-        toast.success(intl.formatMessage(profileMessages.userUpdated));
-      })
-      .catch(() => {
+    onUpdate(data).then((res) => {
+      if (res) toast.success(intl.formatMessage(profileMessages.userUpdated));
+      else {
```

Comment:
For the purpose of preventing the below

![gif](http://g.recordit.co/U5KESOC3cM.gif)